dashboard/config/database.yml

```diff
@@ -29,10 +31,10 @@ mysql_defaults: &mysql_defaults
   # `MYSQL_OPT_CONNECT_TIMEOUT`: The connect timeout in seconds.
   connect_timeout: 2
 
-  # `MYSQL_OPT_RECONNECT`: Enable or disable automatic reconnection to the server if the connection is found to have
-  # been lost. Reconnect is off by default; this option provides a way to set reconnection behavior explicitly.
-  # See https://dev.mysql.com/doc/refman/5.7/en/c-api-auto-reconnect.html
+<% if DCDO.get('enable_mysql_client_reconnect', true) %>
+  # `MYSQL_OPT_RECONNECT` is deprecated https://dev.mysql.com/doc/c-api/8.0/en/c-api-auto-reconnect.html
   reconnect: true
+<% end %>
```

Comment:
This only gets executed at service start, correct? So it should only apply to new EC2 instances launched after the DCDO change - unless any running servers happen to restart their dashboard/pegasus services, right?

Reply:
Correct.